### PR TITLE
Add ability to drop individual points from season total

### PR DESCRIPTION
**Pull request by johnwheffner**
I didn't want to implement the policy logic of dropping the N lowest races, so just annotating by hand and recording in the data model.

I really should plumb the `points_dropped` property through to the view rather than just passing a "(points)" string, but this is easier for now.

### CODE REVIEW

File: app/race/views.py [hunk 1]
```diff
@@ -491,13 +494,16 @@ def edit_participant(race_id, participant_id):
...
         participant.relegated = relegated
+        participant.points_dropped = points_dropped
         participant.disqualified = disqualified
         db.session.commit()
         flash('Racer ' + participant.racer.name + ' updated in race!')
         current_app.logger.info('%s[%d]:%s[%d]', race.name, race.id,
                                 participant.racer.name, participant.id)
         return redirect(url_for('race.details', id=race.id))
 
+    print('Points dropped: %s' % str(participant.points_dropped))
```
Comment (jyundt) [inline app/race/views.py]: Is this for debugging?

File: app/main/views.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def _gen_ind_standings(race_info, race_calendar):
 
     # A list of per-race points for each racer
     racer_race_points = {
-        racer_id: list((ri.points, ri.race_date) for ri in g)
+        racer_id: list((ri.points if not ri.points_dropped else '(%d)' % ri.points or 0, ri.race_date) for ri in g)
```
Comment (jyundt) [inline app/main/views.py:114]: Any thoughts on doing something similar for the Race view? Or maybe some other way in the Race view to denote that points have been dropped?

**Review by jyundt — APPROVED** [review]
Two minor comments, but other than that LGTM.

Thank you for taking care of this.

File: app/main/views.py (hunk 2 above)
Comment (johnwheffner) [reply app/main/views.py:114]: Maybe, though I feel like it's not terribly interesting in that context -- it really only matters for overall season points, or if you're the person setting the dropped bit.

**Pull request merged**